src/page/ProductDetail.page.ts

```diff
@@ -2,7 +2,7 @@ import { $, ElementFinder, promise } from 'protractor';
 
 export class ProductDetailPage {
   private get cartMenu(): ElementFinder {
-    return $('#add_to_cart > button > span');
+    return $('[name="Submit"] > span');
```

Comment:
this change is not part of the commit changes, it should go in a separate commit